tests/test_mail/test_base.py

```diff
@@ -301,9 +300,12 @@ def test_envelope_skill_id():
     assert envelope.skill_id == PublicId("author", "skill_name", "0.1.0")
 
 
-def test_envelope_skill_id_raises_value_error(caplog):
+def test_envelope_skill_id_raises_value_error():
     """Test the property Envelope.skill_id raises ValueError if the URI is not a public id.."""
-    with caplog.at_level(logging.DEBUG, logger="aea.mail.base"):
+    with unittest.mock.patch.object(
+        aea.mail.base.logger, "debug"
+    ) as mock_logger_method:
+        # with caplog.at_level(logging.DEBUG, logger="aea.mail.base"):
```

Comment:
kept this commented for future reference. Maybe better add an explanatory comment.